src/components/add-movie-form/AddMovieForm.jsx

```diff
@@ -0,0 +1,39 @@
+import { useRouteLoaderData } from "react-router-dom";
+import Dialog from "../dialog/Dialog";
+import MovieForm from "../movie-form/MovieForm";
+import { useNavigate, useRevalidator } from "react-router-dom";
+import { useContext } from "react";
+import { ApiContext } from "../..";
```

Comment:
'../..' might cause cyclic dependency, ApiContext might be moved out of index.js